src/main/java/org/sagebionetworks/bridge/udd/dynamodb/DynamoHelper.java

```diff
@@ -86,12 +95,33 @@ public StudyInfo getStudy(String studyId) {
     }
 
     /**
+     * Gets the set of survey table IDs for a given study.
+     *
+     * @param studyId
+     *         ID of study to get survey tables
+     * @return set of survey table IDs, may be empty, but will never be null
+     */
+    public Set<String> getSynapseSurveyTablesForStudy(String studyId) {
+        Item item = ddbSynapseSurveyTable.getItem("studyId", studyId);
+        if (item == null) {
+            return ImmutableSet.of();
```

Comment:
Performance-wise, ImmutableSet is a highly-optimized Guava library, so the performace impact of calling ImmutableSet.of() is negligible.

Code-clarity-wise, ImmtuableSet.of() is already constant-like, so it's prolly not worth making a constant out of this.